assets/home/sections/get-help.js

```diff
@@ -1,22 +1,110 @@
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+
 /**
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
+import { Icon, help, lock, external } from '@wordpress/icons';
 
 /**
  * Internal dependencies
  */
 import Section from '../section';
 
+/**
+ * Component representing each of the items under a Help Category.
+ *
+ * @param {Object}      props           Component properties.
+ * @param {string}      props.title     The title.
+ * @param {string|null} props.url       Optional url. When missing the title will not be clickable.
+ * @param {Object|null} props.extraLink An extra link that will be displayed next to the main title.
+ */
+const Item = ( { title, url, extraLink } ) => {
+	const isTitleInteractive = url !== null;
+
+	const link = isTitleInteractive ? (
+		<a href={ url } target="_blank" rel="noreferrer">
+			{ title }
+		</a>
+	) : (
+		title
+	);
+
+	return (
+		<li className="sensei-home__help-item">
+			<div className="sensei-home__help-item__icon">
+				<Icon icon={ isTitleInteractive ? help : lock } size={ 24 } />
+			</div>
+			<div
+				className={ classNames( 'sensei-home__help-item__title', {
+					'sensei-home__help-item__title--disabled': ! isTitleInteractive,
+				} ) }
+			>
+				{ link }
+			</div>
+			{ extraLink && (
+				<div className="sensei-home__help-item__extra-link">
+					<a href={ extraLink.url } target="_blank" rel="noreferrer">
+						{ extraLink.label }{ ' ' }
+						<Icon icon={ external } size={ 16 } />
```

Comment:
Since it's something generic, I think we could check the URL to add the external icon or not. Maybe it could be an external component that we could use for the Quick Links too? Or did you think something different to handle it there?

Reply:
Done here 10517d394